_pages/guides/data-hub-framework.markdown

```diff
@@ -5,17 +5,110 @@ lead_text: ''
 permalink: /guides/data-hub-framework/
 ---
 
-We often want to run Grove as a UI working alongside a [MarkLogic Data Hub Framework (DHF)](https://marklogic.github.io/marklogic-data-hub/). This page demonstrates how to do that.
+# Grove + MarkLogic Data Hub
 
-# Recommended Method to Run Grove with DHF
+We often want to run Grove as a UI working alongside a [MarkLogic Data Hub](https://marklogic.github.io/marklogic-data-hub/). This guide provides the mechanics on how to do this, and provides a choice of method.
 
-In order to run a Grove UI project alongside the MarkLogic Data Hub Framework (DHF) project, we recommend creating a Grove project-specific app-server, which has its own modules database but points at the content database with the data you wish to visualize: most often the DHF project's FINAL database. The DHF project will be responsible for managing the content database (including setting up indexes - needed for facets - and security permissions), as well as the content database's related triggers and schemas databases, while the Grove project will manage the Grove project's specific app-server and modules database.
+This guide has been vetted on both Data Hub 5.0 and the Data Hub Framework 4.3.1. 
+
+## Migrating Grove-related artifacts and code to the Data Hub
+
+In order to run a Grove UI project alongside a MarkLogic Data Hub, we recommend migrating Grove related artifacts and code from its code base into your Data Hub project.  
```

Comment:
"This approach is tested against both DHF and DHS"

Reply:
Added note.